tests/test_pe_reports.py

```diff
@@ -23,10 +23,11 @@
 PROJECT_VERSION = pe_reports.__version__
 
 
-# TODO: Replace current dummy test with useful tests - https://github.com/cisagov/pe-reports/issues/3#issue-909531010
+# TODO: Replace current dummy test with useful tests
+# Issue - https://github.com/cisagov/pe-reports/issues/3#issue-909531010
```

Comment:
Can this TODO be removed now that there are better tests here?

Reply:
@dav3r, actually this one is in reference to just the` pe-report` module,  not the `pe-mailer` module. We do have intention to add in relevant tests once we get the `develop `branch updated with our new 1.0.0 report.

Reply:
Got it- thanks for the clarification.